test/system/roles/public_health/dashboard/export_verifier.rb

```diff
@@ -236,6 +240,21 @@ def verify_full_history_export(xlsx_monitorees, xlsx_assessments, xlsx_lab_resul
       end
     end
 
+    exported_vaccines = xlsx_vaccines.sheet('Vaccinations')
+    vaccines = Vaccine.where(patient_id: patient_ids).order(:patient_id)
+    assert_equal(vaccines.size, exported_vaccines.last_row - 1, 'Number of results in Vaccinations')
+    vaccine_headers = ['Patient ID', 'Vaccine Group', 'Product Name', 'Administration Date', 'Dose Number', 'Notes', 'Created At', 'Updated At']
+    vaccine_headers.each_with_index do |header, col|
+      assert_equal(header, exported_vaccines.cell(1, col + 1), "For header: #{header} in Vaccinations")
+    end
+    vaccines.each_with_index do |vaccine, row|
+      details = vaccine.attributes.except('id')
```

Comment:
this is awesome, would you mind changing `details` for labs and histories to this and removing both `details` methods from their classes since they're not being used anywhere else?